content/en/docs/components/model-registry/getting-started.md

```diff
@@ -57,15 +58,15 @@ You can use the `register_model` method to index a model's artifacts and its met
 
 ```python
 rm = registry.register_model(
-    "mnist",
-    "https://github.com/tarilabs/demo20231212/raw/main/v1.nb20231206162408/mnist.onnx",
-    model_format_name="onnx",
+    "iris",
+    "gs://kfserving-examples/models/sklearn/1.0/model",
```

Comment:
Does this require login access to download?

If so, I would use one of the model.joblib I've used in another demo with link to github

Reply:
no, at the time of writing no authorization is required to fetch the model with kserve, it's a public bucket